geoh5py/data/reference_value_map.py

```diff
@@ -66,7 +66,7 @@ def validate_value_map(cls, value_map: np.ndarray | dict) -> np.ndarray:
             if value_map.ndim == 1:
                 unique_set = set(value_map)
                 unique_set.discard(0)
-                value_map = {i + 1: str(val) for i, val in enumerate(unique_set)}
+                value_map = {int(val): f"Unit {val}" for val in unique_set}
```

Comment:
How can you ensure value map can be tranfer to int?
I don't get how 1: "unit 1" is useful for value_map?